packages/calcite-components/src/components/dialog/dialog.e2e.ts

```diff
@@ -716,17 +716,15 @@ describe("calcite-dialog", () => {
     const dialog = await page.find("calcite-dialog");
     await page.waitForChanges();
     expect(dialog).toHaveAttribute("open");
-    expect(dialog).toHaveAttribute("open");
+    await page.waitForChanges();
```

Comment:
Is this necessary? There's a previous call to `waitForChanges` and no prop is modified before this call.

Reply:
I had to add a timeout there and some other places, otherwise tests fail.